code/modules/mob/living/simple_animal/bot/secbot.dm

```diff
@@ -240,7 +240,7 @@
 		speak("[arrest_type ? "Detaining" : "Arresting"] level [threat] scumbag <b>[C]</b> in [location].", radio_channel)
 
 	var/armor_block = C.run_armor_check(BODY_ZONE_CHEST, "stamina")
-	C.apply_damage(85, STAMINA, BODY_ZONE_CHEST, armor_block)
+	C.apply_damage(45, STAMINA, BODY_ZONE_CHEST, armor_block)
```

Comment:
```suggestion
	C.apply_damage(60, STAMINA, BODY_ZONE_CHEST, armor_block)
```

Make it match the actual stun baton for consistency. 60 is still a possible 2-hit stamcrit, but with less immediate slowdown and it's possible to be 3-hit with some armor.